srcpkgs/cli-visualizer/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'cli-visualizer'
+pkgname=cli-visualizer
+version=1.7
+revision=1
+build_style=gnu-configure
+build_options="pulseaudio"
+build_options_default="pulseaudio"
+make_install_args="PREFIX=${DESTDIR}/usr/bin/"
+hostmakedepends="clang"
+makedepends="fftw-devel ncurses-devel
+ $(vopt_if pulseaudio 'pulseaudio-devel')"
+depends="libfftw ncurses"
+short_desc="CLI based audio visualizer"
+maintainer="Jasper Chan <jasperchan515@gmail.com>"
+license="MIT"
+homepage="https://github.com/dpayne/cli-visualizer"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=0445fbfad9707668a70ff146f3f24308f28fa0ae67a7010bfdea0dacba75b75b
+
+case "$XBPS_TARGET_MACHINE" in
+	armv*) make_build_args+=" VIS_COMPILER_ARCH=armv6" ;;
+esac
+
+pre_install() {
+	mkdir -p "${DESTDIR}/usr/bin/"
+}
+
+post_install() {
+	# Files listed in install.sh
+	vsconf examples/config
+
+	# Move color files (this is how they should be in the config dir)
```

Comment:
I took a serious look at the package and the install.sh is meant to run as a normal user, not root.

the config dir mentioned refers to the config dir in the user's $HOME.

we don't touch that, the package manager mostly touches /usr.

Reply:
Yea I know, I just wanted to put them like that because it's not entirely obvious to an end user installing without ever looking at the repo where those files should go.